README.md

```diff
@@ -33,11 +36,11 @@ The `/app` directory is created and set as the current working directory. This i
 Running a built-in worker class without a config file:
 ```shell
 docker run --rm -it \
-  -e WORKER_CLASS=vumi.blinkenlights.MetricTimeBucket \
   -e AMQP_HOST=rabbitmq.service.consul \
   -e VUMI_OPT_BUCKETS=3 \
   -e VUMI_OPT_BUCKET_SIZE=10 \
-  praekeltfoundation/vumi
+  praekeltfoundation/vumi \
+    --worker-class vumi.blinkenlights.MetricTimeBucket
```

Comment:
This is a change to the example to show mixing environment variables and command line options. The old example still works.